src/haz3lweb/explainthis/data/TerminalExp.re

```diff
@@ -10,6 +10,15 @@ let triv_exp: form = {
 };
 let triv_exps: group = {id: TrivExp, forms: [triv_exp]};
 
+let undefined_exp: form = {
+  id: UndefinedExp,
+  syntactic_form: [exp("Undefined")],
+  expandable_id: None,
+  explanation: "Undefined.",
```

Comment:
I wonder if you could explain this a little more, and maybe add an example of an incomplete case statement, or factorial only defined on positives or something?

Reply:
Updated the documentation with two new examples.